subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3152,26 +3152,26 @@ static inline void dle_max_time_get(const struct ll_conn *conn,
 {
 	u16_t rx_time = 0;
 	u16_t tx_time = 0;
-	u16_t bit = 0;
+	u8_t  phy_bitmask = 0;
+
+	if (!conn->common.fex_valid) {
+		phy_bitmask = 0;
```

Comment:
There were some other changes in the master branch
ref #23482 and #23557
In #23557 there are defines for the BIT(x) statements; rebase on master and use those, and replace 0 with the appropriate define as well (PHY_1M)